docs/source/configuration/hooks.rst

```diff
@@ -68,9 +68,14 @@ Apart from command pre- and posthooks, the following hooks will be interpreted:
     :type timestamp: :obj:`datetime.datetime`
     :rtype: string
 
+.. _pre-edit-translate:
+
 .. py:function:: pre_edit_translate(bodytext[, ui= None, dbm=None])
 
-    used to manipulate a messages bodytext *before* the editor is called.
+    Used to manipulate a messages bodytext *before* the editor is called.  The
+    bodytext might also contain some header lines, depending on the settings
```

Comment:
I'd replace all mentionings of *body*text in order to avoid confusion here. How about "message text"?
Apart from this it looks good to me.